src/Microsoft.DotNet.VersionTools/Util/Command.cs

```diff
@@ -0,0 +1,348 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.VersionTools.Util
+{
+    public class Command
```

Comment:
I think all these "Utils" should probably be internal. We don't want external callers using these.  (It isn't the job of the "VersionTools" to expose these classes)